packages/untp-test-suite/src/templates/mapper.ts

```diff
@@ -0,0 +1,28 @@
+import Handlebars from 'handlebars';
+import fs from 'fs/promises';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+import { TestSuiteResult } from '../core/types/index.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+Handlebars.registerHelper('jsonArray', (array: object) => {
+  return JSON.stringify(array);
+});
```

Comment:
the name of the helper is not good. We should make general like `jsonStringify` because the input is an object. Furthermore, should we validate the input parameter?